src/router/legacyPermalinkManagement.routerPlugin.js

```diff
@@ -176,7 +176,11 @@ const handleLegacyParams = (legacyParams, store, to, next) => {
     const urlWithoutQueryParam = window.location.href.substr(0, window.location.href.indexOf('?'))
     window.history.replaceState(window.history.state, document.title, urlWithoutQueryParam)
 
-    if ('adminId' in legacyParams) {
+    // TODO BGDIINF_SB-2685: re-activate once on prod
+    //  we cannot let adminId get through right now because legacy KML will be broken by the new viewer if edited
+    //  (broken in a sense that they will not be usable by mf-geoadmin3 anymore)
+    const absolutelyNot = () => false
+    if (absolutelyNot() && 'adminId' in legacyParams) {
```

Comment:
Would be actually better to check the hostname, if `map.geo.admin.ch` then true, otherwise false, this way we have less risk forgetting this during the go live.

Reply:
but if it doesn't fail, we might forget to do it, don't you think?